src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_FisherStrandUnitTest.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator.allelespecific;
+
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by emeryj on 8/11/17.
+ */
+public class AS_FisherStrandUnitTest extends ReducibleAnnotationBaseTest {
```

Comment:
Can you make a note that this is used in the gatk3 concordance test?  Similar for MQRankSum.

Reply:
done